README.md

```diff
@@ -52,5 +52,37 @@ You can alter the environment variables to use different sample files and timing
 | `BUILD`     | A boolean (`true` or not set string) to toggle the build and push of the performance tests as a Docker image.                                                                       | `IMAGE=true`                                                                 | None                                                             | no       |
 | `NAMESPACE` | The k8s namespace to run the performance tests as a pod in.                                                                                                                         | `NAMESPACE=rm`                                                               | None                                                             | no       |
 
+### Run RabbitMQ Performance testing
+
+Run
+```bash
+make rabbitmq-perf-test
+docker push eu.gcr.io/census-rm-ci/rm/census-rm-rabbit-performance
+```
+Set Kubectl to census-rm-performance and then apply the deployment
+```bash
+gcloud beta container clusters get-credentials rm-k8s-cluster --region <REGION> --project <PERFORMANCE_PROJECT_NAME>
+kubectl apply -f rabbit-perf-test/census-rm-rabbit-perf-deployment.yml
+```
+
+Shell into pod to run a basic test.
+This test is continuous so you need to 'CTRL C' to stop it.
+```bash
+./basic-test.sh
```

Comment:
Needs to be updated to the tests directory `./tests/basic-test.sh`

Reply:
Done.

Reply:
Hasn't been done!

Reply:
Pushed now!